### DISCUSSION TITLE
[RFC] Some Questions about stash

### DISCUSSION THREAD

**Discussion by kucingkembar** [stashapp/stash#4978] [Q&A]
sorry for my bad English and if I do not in the normal pattern/form/template way,
I have some questions :
1. is possible to run 2 separate Stash programs on 1 pc?
2. Are there any other end-points besides theporndb.net & stashdb.org?
3. is there a way to remove "Play Count"?
4. is there a way to change the priority on the Task Queue without canceling the current Task?

thank you for reading and have a nice day

**Comment by WithoutPants** [comment]
1. Yes. They will need to run on different ports and from different working directories. You can edit the port in `config.yml`.
2. Yes, I don't know the specific names or URLs though. Someone else may provide an answer.
3. You can remove or reset play counts in the History tab of the scene page. You can disable the auto-increment of play counts in the Interface settings.
4. Not currently.

**Comment by DogmaDragon** [comment]
## Known stash-box Instances

- [StashDB](<https://stashdb.org>) - focused on mainstream porn studios.
  - [StashDB contribution guidelines](<https://guidelines.stashdb.org/>)
  - [Accessing StashDB](<https://guidelines.stashdb.org/docs/faq_getting-started/stashdb/accessing-stashdb/>)
  - [Correction backlog](<https://docs.google.com/spreadsheets/d/1eiOC-wbqbaK8Zp32hjF8YmaKql_aH-yeGLmvHP1oBKQ/edit?usp=sharing>) *(for experienced users)*
- [FansDB](<https://fansdb.cc>) - focused on individual adult content creators.
  - [Accessing FansDB](<https://docs.fansdb.cc/#accessing-fansdb>)
  - [FansDB guidelines](<https://docs.fansdb.cc/#guidelines>)
  - [FansDB Discord](<https://discord.gg/dhJaFAgYAj>)
- [PMV Stash](<https://pmvstash.org>) - focused on remixed pornographic art such as Porn Music Videos, Cock Hero, Hypno Vids, Compilations.
  - [PMV Stash Discord](<https://discord.gg/qGbs8kvpVk>)
  - Invites are distributed via their Discord.
- [ThePornDB](<https://theporndb.net>) - focused on mainstream porn studios. 
  - [ThePornDB Discord](<https://discord.com/invite/XpSGpaB>)
  - [Accessing ThePornDB](<https://theporndb.net/tools>) *(requires registration to see Stash TPDB Endpoint)*
